message_sender/factory.py

```diff
@@ -59,9 +79,60 @@ def _raw_send(self, py_data):
         res = r.json()
         return res.get('result', {})
 
+    def _override_payload(self, payload):
+        if self.override_payload:
+            old_payload = payload
+
+            def get_value(data, key):
+                index = key.find('.')
+                if index != -1:
+                    nested_data = data.get(key[:index], {})
+                    nested_key = key[index+1:]
+
+                    return get_value(nested_data, nested_key)
+
+                return data.get(key, key)
```

Comment:
I'm wondering if `key` is a good default value for this? Maybe `None` will be a better default value, because then it's explicit that that key doesn't have a value.

Reply:
I made `key` default for cases where the key is the actual string you want to send.

Reply:
Ah, I see. So when you need to send a static thing. Yes that makes sense, leave it as it is.